src/handlebars/installers.handlebars

```diff
@@ -0,0 +1,169 @@
+{{> header title='Installers | ' style1='<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/styles/obsidian.min.css">' }}
+
+<main class="grey-bg">
+
+  <h1 class="large-title">Installers</h1>
+
+  <div class="align-left support">
+    <div class="margin-bottom">
+      <p>Installers are currently available for Windows and macOS JDK and JRE packages.</p>
+
+    </div>
+
+    <div class="margin-bottom">
+
+      <h3 id="windows-msi">Windows MSI packages</h3>
+
+        <p>AdoptOpenJDK Windows installer packages are available as standard <tt>.msi</tt> files, which can be run with an interactive user interface or run silently
+          from the command line. In addition, you can use a capability like Active Directory Group Policy Object (GPO) to distribute the packages to target workstations
```

Comment:
Active Directory deployment is working ? CF https://github.com/AdoptOpenJDK/openjdk-installer/issues/57